extension/content_script.js

```diff
@@ -298,12 +298,14 @@ function highlightKeyWords(container, start, end) {
 	let wordRegex = /\b\w+\b/g;
 	let wordList = sentenceText.match(wordRegex);
 	let keywords = doc.getKeyWords();
+	// Look where keyword is in sentence AFTER last search. Guaranteed to be after. Init to 0. 
+	let keyword_search_start_pointer = start; 
 	for (var i in wordList) { // Accentuate keywords
-		// TODO: This only gets the first occurence of each word in the sentence; should get all
 		let word = wordList[i];
 		if (keywords.has(word.toLowerCase())) { // See if each word is a keyword
-			var word_start = containerText.indexOf(word, start);
+			var word_start = containerText.indexOf(word, keyword_search_start_pointer);
 			var word_len = word.length;
+			keyword_search_start_pointer = word_start + word_len;
```

Comment:
nit: save a line and just do + word.length

Reply:
We use word_len below as well: 

`ontainer.markRanges([{ 
			start: word_start,
			length: word_len
		}], {`

Reply:
Ehh I can still get rid of it though, it's self explanatory